Makefile

```diff
@@ -43,3 +43,8 @@ stop-dummy-host:
 start-all: start-zabbix-server start-rundeck start-dummy-host
 
 stop-all: stop-zabbix-server stop-rundeck stop-dummy-host
+
+test:
+	go test ./...
```

Comment:
How about linting before running tests?

Reply:
I'm fine with either. Do you prefer we switch to linting before running tests?

Reply:
Done and updated to skip vendor when linting. Not sure of a better way to do this.